### ISSUE TITLE
Add show / hide option to show in status bar full path of selected node

### ISSUE THREAD

**Issue by ratra1** [giuspen/cherrytree#2525]
Can you please add show / hide option to show in status bar full path of selected node.

So for selected node in status bar to be shown something like: root/node_1/sub_node_of_node_1/...

**Comment by giuspen** [comment]
But the full path to the selected node is also shown on the node name header, isn't that enough?

![image](https://github.com/giuspen/cherrytree/assets/6842059/5027c087-3831-429a-a9f1-8c7098ffc40b)

**Comment by ratra1** [comment]
> But the full path to the selected node is also shown on the node name header, isn't that enough?
> 
Yes. But I would like to see the path to selected node when Tree Explorer is hidden. Anyway the app is great as it is and this is just a suggestion for future.

**Comment by gitvectors** [comment]
"But the full path to the selected node is also shown on the node name header, isn't that enough?"

Consider a CodeBox _inside_ the node where a Python (or other) script requires to parse the content of that node. How does the Python script refer to that node?

**Comment by giuspen** [comment]
> > But the full path to the selected node is also shown on the node name header, isn't that enough?
> 
> Yes. But I would like to see the path to selected node when Tree Explorer is hidden. Anyway the app is great as it is and this is just a suggestion for future.

But the node name header with the full path to node is not in the tree explorer, it is above the text, look at the top right in the screenshot edxample for `b / sh / html`

**Comment by giuspen** [comment]
> "But the full path to the selected node is also shown on the node name header, isn't that enough?"
> 
> Consider a CodeBox _inside_ the node where a Python (or other) script requires to parse the content of that node. How does the Python script refer to that node?

This is a different matter, you would like to support code executed in a node with for example environmental variables that expose node name or else... it is a fair request but this should go to another ticket along with description of a practical case / test example
